pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, libelf, libcap, libseccomp
+, nvidia_x11 }:
+
+with lib; let
+
+  persistenced = nvidia_x11.passthru.persistenced;
+
+in stdenv.mkDerivation rec {
+  name = "libnvidia-container-${version}";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "libnvidia-container";
+    rev = "v1.0.0";
+    sha256 = "1ws6mfsbgxhzlb5w1r8qqg2arvxkr21n59i4cqsyz3h5jsqsflbw";
+  };
+
+  nvidia-modprobe = fetchurl {
+    url = "https://github.com/NVIDIA/nvidia-modprobe/archive/396.51.tar.gz";
```

Comment:
No need to define this as an attribute to `mkDerivation`, define this in the `let in` above instead.